api_tester/lib/api_tests/models/store_transaction_api_test.dart

```diff
@@ -8,29 +8,22 @@ class _StoreTransactionApiTest {
     StoreTransaction transaction = StoreTransaction.fromJson(json);
   }
 
-  void _checkToJson(StoreTransaction transaction) {
-    Map<String, dynamic> json = transaction.toJson();
-  }
-
   void _checkConstructor(
       String transactionIdentifier,
-      String revenueCatIdentifier,
       String productIdentifier,
       String purchaseDate) {
     StoreTransaction transaction =
-        StoreTransaction(revenueCatIdentifier, productIdentifier, purchaseDate);
```

Comment:
This is another breaking change... I could have kept `revenueCatIdentifier` which was deprecated, but would mean additional mapping code, and I felt it would be ok to remove it now.